letsencrypt-auto

```diff
@@ -19,7 +19,7 @@ XDG_DATA_HOME=${XDG_DATA_HOME:-~/.local/share}
 VENV_NAME="letsencrypt"
 VENV_PATH=${VENV_PATH:-"$XDG_DATA_HOME/$VENV_NAME"}
 VENV_BIN="$VENV_PATH/bin"
-LE_AUTO_VERSION="0.7.0"
+LE_AUTO_VERSION="0.8.0"
```

Comment:
This PR shouldn't be changing `letsencrypt-auto`, though maybe if you merge master into your branch this will go away?